lib/utils/blockchain-txs.ts

```diff
@@ -108,24 +108,14 @@ export async function ccipRequest({
   body,
   url,
 }: CcipRequestParameters): Promise<Response> {
-  return new Promise((resolve, reject) => {
-    return fetch(url.replace("/{sender}/{data}.json", ""), {
-      body: JSON.stringify(body, (_, value) =>
-        typeof value === "bigint" ? value.toString() : value
-      ),
-      method: "POST",
-      headers: {
-        "Content-Type": "application/json",
-      },
-    })
-      .then((res) => {
-        return res.json();
-      })
-      .then((res) => resolve(res))
-      .catch((err) => {
-        console.error(err);
-        reject(err);
-      });
+  return fetch(url.replace("/{sender}/{data}.json", ""), {
```

Comment:
now you need to handle the exception everywhere you call this function because you removed the inline `catch` statement, just fyi